package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bpmn-auto-layout",
-  "version": "0.1.0",
```

Comment:
Let's discard changes in package(-lock).json as this will be done automatically once we release the library.